test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/q_and_mq_as_uri_param_for_metadata.test

```diff
@@ -529,10 +529,6 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
                     "M1": {
                         "type": "Number", 
                         "value": 3
-                    }, 
```

Comment:
This change in existing .test is due to the bugfix of issue #3310

NTC (informative)